### ISSUE TITLE
[torchbench] resnet50_quantized_qat fails to run 

### ISSUE THREAD

**Issue by alexbaden** [intel/intel-xpu-backend-for-triton#458]
```
 » benchmarks/dynamo/torchbench.py --float32 -dxpu -n10 --no-skip --dashboard --training --inductor --accuracy --output /tmp/torchbench.csv --filter resnet50_quantized_qat




loading model: 0it [00:03, ?it/s]
xpu  train resnet50_quantized_qat             
WARNING:common:fp64 golden ref were not generated for resnet50_quantized_qat. Setting accuracy check to cosine
/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/fbgemm_gpu/fbgemm_gpu_py.so: undefined symbol: _ZNK5torch8autograd4Node4nameEv
/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/fbgemm_gpu/fbgemm_gpu_py.so: undefined symbol: _ZNK5torch8autograd4Node4nameEv
skipping cudagraphs for unknown reason
ERROR:common:output with shape [1] doesn't match the broadcast shape [64]
Traceback (most recent call last):
  File "/localdisk/abaden/Projects/frameworks.ai.pytorch.private-gpu/benchmarks/dynamo/common.py", line 2145, in check_accuracy
    new_result = optimized_model_iter_fn(model_copy, example_inputs)
  File "/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/torch/_dynamo/eval_frame.py", line 328, in _fn
    return fn(*args, **kwargs)
  File "/localdisk/abaden/Projects/frameworks.ai.pytorch.private-gpu/benchmarks/dynamo/common.py", line 1909, in run_n_iterations
    self.model_iter_fn(mod, inputs, collect_outputs=False)
  File "/localdisk/abaden/Projects/frameworks.ai.pytorch.private-gpu/benchmarks/dynamo/torchbench.py", line 461, in forward_and_backward_pass
    cloned_inputs = clone_inputs(inputs)
  File "/localdisk/abaden/Projects/frameworks.ai.pytorch.private-gpu/benchmarks/dynamo/torchbench.py", line 462, in <resume in forward_and_backward_pass>
    self.optimizer_zero_grad(mod)
  File "/localdisk/abaden/Projects/frameworks.ai.pytorch.private-gpu/benchmarks/dynamo/torchbench.py", line 462, in <resume in forward_and_backward_pass>
    self.optimizer_zero_grad(mod)
  File "/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/torch/_dynamo/eval_frame.py", line 328, in _fn
    return fn(*args, **kwargs)
  File "/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/torch/_dynamo/external_utils.py", line 17, in inner
    return fn(*args, **kwargs)
  File "/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/torch/_functorch/aot_autograd.py", line 3905, in forward
    return compiled_fn(full_args)
  File "/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/torch/_functorch/aot_autograd.py", line 1482, in g
    return f(*args)
  File "/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/torch/_functorch/aot_autograd.py", line 2624, in runtime_wrapper
    original_inpt.copy_(updated_inpt)
RuntimeError: output with shape [1] doesn't match the broadcast shape [64]
```

Similar to #454

**Comment by etiotto** [comment]
@alexbaden I see  these warnings. Do you know how to fix them ? 
```
WARNING:common:fp64 golden ref were not generated for resnet50_quantized_qat. Setting accuracy check to cosine
/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/fbgemm_gpu/fbgemm_gpu_py.so: undefined symbol: _ZNK5torch8autograd4Node4nameEv
/localdisk/abaden/Projects/envs/triton-benchmark-env/lib/python3.10/site-packages/fbgemm_gpu/fbgemm_gpu_py.so: undefined symbol: _ZNK5torch8autograd4Node4nameEv
skipping cudagraphs for unknown reason
```

**Comment by alexbaden** [comment]
Nope - it is not clear if there is an Intel FBGEMM XPU port or if there are workarounds for fbgemm calls in IPEX.

**Comment by whitneywhtsang** [comment]
Also fail with v2.1.

**Comment by ienkovich** [comment]
The error is the same for both CPU and XPU when TorchInductor is used:
```
RuntimeError: output with shape [1] doesn't match the broadcast shape [64]
```
So the problem is not related to the Tirton XPU backend.

**Comment by vlad-penkin** [comment]
CUDA has the same failure that will be resolved in PyTorch 2.3. Not impact on PT Upstream to reach on par pass rate with CUDA.
